packages/router/src/grpc/view-protocol-server/ephemeral-address.ts

```diff
@@ -1,9 +1,9 @@
-import type { Impl } from '.';
+import type { ViewImpl } from '.';
 import { servicesCtx } from '../../ctx';
 
 import { getEphemeralByIndex } from '@penumbra-zone/wasm';
 
-export const ephemeralAddress: Impl['ephemeralAddress'] = async (req, ctx) => {
+export const ephemeralAddress: ViewImpl['ephemeralAddress'] = async (req, ctx) => {
```

Comment:
Easier to re-export in barrel file if they aren't all named the same `Impl` var

Reply:
the type is for local use, so we should probably break this out into another file that isn't exported from the package

also, now that we're no longer doing barrel this change could revert